src/main/java/org/ays/common/model/response/AysErrorResponse.java

```diff
@@ -105,7 +105,7 @@ public enum Header {
         /**
          * NOT_FOUND header.
          */
-        NOT_FOUND("NOT EXIST"),
+        NOT_FOUND("NOT EXIST ERROR"),
```

Comment:
Not Found yazan bütün yerleri Not Exist olarak değiştirmemiz gerekiyor
```suggestion
        NOT_EXIST_ERROR("NOT EXIST ERROR"),
```

Reply:
HttpStatus classına "NOT_EXIST_ERROR" eklemeli miyim?

Reply:
@yagmurbarank `HttpStatus` sınıfı Spring'in kendi sınıfı olan sınıftan bahsediyorsan orayı değiştiremiyoruz. Bu sebeple orası aynı şekilde devam edecek. Biz kendi döndüğümüz header bilgisini değiştiriyor olacağız